Resources/Locale/en-US/accent/mobster.ftl

```diff
@@ -95,7 +95,7 @@ accent-mobster-words-29 = gun
 accent-mobster-words-replace-29 = gat
 
 accent-mobster-words-30 = boss
-accent-mobster-words-replace-30 = don
```

Comment:
wait actually why are you dropping don for boss?

Reply:
Because they say boss in other places, and mobsters saying boss is more iconic than saying don, imo

Reply:
fair enough!